Microsoft.Toolkit.Uwp.UI.Controls.Core/TextToolbar/TextToolbar.Events.cs

```diff
@@ -248,43 +248,57 @@ private void Editor_KeyDown(object sender, KeyRoutedEventArgs e)
 
             LastKeyPress = e.Key;
 
-            if (GetTemplateChild(RootControl) is CommandBar root)
+            if (GetTemplateChild(RootControl) is CommandBar root && IsDoingShortcut(e.Key))
```

Comment:
Only this line changed here, the following lines have not been touched but the indentation changed.